include/ofi_util.h

```diff
@@ -77,25 +77,29 @@
 		(entry).err,strerror((q), (entry).prov_errno, (entry).err_data, NULL, 0),	\
 		(entry).prov_errno)
 
-#define OFI_Q_READERR(prov, log, q, q_str, readerr, strerror, ret, err_entry)	\
-	do {									\
-		(ret) = readerr((q), &(err_entry), 0);				\
-		if ((ret) != sizeof(err_entry)) {				\
-			FI_WARN(prov, log,					\
-				"Unable to fi_" q_str "_readerr\n");		\
-		} else {							\
-			OFI_Q_STRERROR(prov, log, q, q_str,			\
-				       err_entry, strerror);			\
-		}								\
+#define OFI_CQ_READERR(prov, log, cq, ret, err_entry)			\
+	do {								\
+		(ret) = fi_cq_readerr((cq), &(err_entry), 0);		\
+		if ((ret) < 0) {					\
+			FI_WARN(prov, log,				\
+				"Unable to fi_cq_readerr: %zd\n", ret);	\
+		} else {						\
+			OFI_Q_STRERROR(prov, log, cq, "cq",		\
+				       err_entry, fi_cq_strerror);	\
+		}							\
```

Comment:
These macros are only used by the rxm provider and only in one place.  We should remove them and just move the code into rxm.

Reply:
Done.